kos_sim/eval_model.py

```diff
@@ -0,0 +1,352 @@
+"""Real robot walking deployment script using PyKOS interface."""
+
+import argparse
+import asyncio
+import math
+import time
+from copy import deepcopy
+from typing import Dict, List, Tuple, Union
+import csv
+
+import numpy as np
+import pygame
+from kinfer.inference.python import ONNXModel
```

Comment:
tbd if it's worth using this now - idk what extra value it is providing, might as well just pass a normal onnx model to avoid adding this as a dependency